openprescribing/measure_definitions/pregabalinddd.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "Prescribing of pregabalin (total DDD)",
+  "title": [
+    "pregabalin DDD per 1000 patients"
+  ],
+  "description": [
+    "Prescribing of pregabalin (total DDD) per 1000 patients"
+  ],
+  "why_it_matters": [
+    "Pregabalin is a relatively new drug, which is increasingly used in neuropathic and other types of pain. ",
+    "Pregabalin costs have reduced recently, and there is still significant use in the NHS. ",
+    "There are <a href='https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/385791/PHE-NHS_England_pregabalin_and_gabapentin_advice_Dec_2014.pdf'>concerns about the potential for abuse</a>. ",
+    "We are using the <a href='https://www.who.int/medicines/regulation/medicines-safety/toolkit_ddd/en/'>",
+    "Defined Daily Dose</a> (DDD) for this measure."
+  ],
+  "numerator_short": "Total pregabalin prescribed (in DDD)",
+  "denominator_short": "1000 patients",
+  "url": null,
+  "is_percentage": false,
+  "is_cost_based": false,
+  "low_is_good": null,
+  "tags": [
+    "core",
+    "pain",
+    "safety"
+  ],
+  "numerator_type": "custom",
+  "numerator_columns": [
+    "SUM(p.quantity * r.strnt)/300 AS numerator, "
```

Comment:
Why divided by 300? Can you add in a SQL comment?

Reply:
It's to calculate the DDD (which is 300).  Added comment